docs/tutorials/intro_tutorial.rst

```diff
@@ -60,19 +60,20 @@ Install Mesa:
 
 .. code:: bash
 
-   python3 -m pip install mesa
+   pip install mesa
 
 Install Jupyter Notebook (optional):
 
 .. code:: bash
 
-   python3 -m pip install jupyter
+   pip install jupyter
 
-Install matplotlib:
+Install Visualization Tools:
 
 .. code:: bash
 
-   python3 -m pip install matplotlib
+   pip install matplotlib
+   pip install seaborn
```

Comment:
I'd stick to just `matplotlib` for now. It doesn't make sense to install `seaborn` when it is not yet used.